runtimes/peregrine/src/runtime_apis.rs

```diff
@@ -475,6 +490,162 @@ impl_runtime_apis! {
 		}
 	}
 
+	impl pallet_bonded_currency_runtime_api::BondedCurrency<Block, Balance, AccountId, Operation, AccountId, FixedPointUnderlyingType, HumanReadablePoolDetails<AccountId, Balance, bonded_currencies::AssetId, AssetId>, BondedCurrencyError> for Runtime {
```

Comment:
I will review this one the rest of the comments have been addressed.

Reply:
Please wait for the review until I re-request one from you. Once we have agreed on the API, I have to change the implementation accordingly